doc/install.txt

```diff
@@ -69,818 +48,44 @@ The following libraries and software are optional:
-
-For more information on the formatting of ``~/.theanorc`` and the
-configuration options that you can put there, see :ref:`libdoc_config`.
-
-Here are some different way to configure BLAS:
-
-0) Do nothing and use the default config, which is to link against the same
-BLAS against which NumPy was built. This does not work in the case NumPy was
-compiled with a static library (e.g. ATLAS is compiled by default only as a
-static library).
-
-1) Disable the usage of BLAS and fall back on NumPy for dot products. To do
-this, set the value of ``blas.ldflags`` as the empty string (ex: ``export
-THEANO_FLAGS=blas.ldflags=``). Depending on the kind of matrix operations your
-Theano code performs, this might slow some things down (vs. linking with BLAS
-directly).
-
-2) You can install the default (reference) version of BLAS if the NumPy version
-(against which Theano links) does not work. If you have root or sudo access in
-fedora you can do ``sudo yum install blas blas-devel``. Under Ubuntu/Debian
-``sudo apt-get install libblas-dev``. Then use the Theano flags
-``blas.ldflags=-lblas``. Note that the default version of blas is not optimized.
-Using an optimized version can give up to 10x speedups in the BLAS functions
-that we use.
-
-3) Install the ATLAS library. ATLAS is an open source optimized version of
-BLAS. You can install a precompiled version on most OSes, but if you're willing
-to invest the time, you can compile it to have a faster version (we have seen
-speed-ups of up to 3x, especially on more recent computers, against the
-precompiled one). On Fedora, ``sudo yum install atlas-devel``. Under Ubuntu,
-``sudo apt-get install libatlas-base-dev libatlas-base`` or
-``libatlas3gf-sse2`` if your CPU supports SSE2 instructions. Then set the
-Theano flags ``blas.ldflags`` to ``-lf77blas -latlas -lgfortran``. Note that
-these flags are sometimes OS-dependent.
-
-4) Use a faster version like MKL, GOTO, ... You are on your own to install it.
-See the doc of that software and set the Theano flags ``blas.ldflags``
-correctly (for example, for MKL this might be ``-lmkl -lguide -lpthread`` or
-``-lmkl_intel_lp64 -lmkl_intel_thread -lmkl_core -lguide -liomp5 -lmkl_mc
--lpthread``).
+On the following pages you will find different installation instructions depending
+on your needs.
 
-.. note::
-
-    Make sure your BLAS
-    libraries are available as dynamically-loadable libraries.
-    ATLAS is often installed only as a static library.  Theano is not able to
-    use this static library. Your ATLAS installation might need to be modified
-    to provide dynamically loadable libraries.  (On Linux this
-    typically means a library whose name ends with .so. On Windows this will be
-    a .dll, and on OS-X it might be either a .dylib or a .so.)
-
-    This might be just a problem with the way Theano passes compilation
-    arguments to g++, but the problem is not fixed yet.
+- :ref:`install_basic`
+- :ref:`install_bleedingedge`
+- :ref:`install_others`
```

Comment:
If I am not wrong there should not be a "Platform-specific" installation instruction.